docs/iidm/model/threeWindingsTransformer.md

```diff
@@ -45,6 +45,7 @@ Three windings transformers can also have [current limits](currentLimits.md) def
 | G | double | S | yes | - | The nominal magnetizing conductance specified at the voltage of the leg |
 | B | double | S | yes | - | The nominal magnetizing susceptance specified at the voltage of the leg |
 | RatedU | double | kV | yes | - | The rated voltage |
+| RatedS | double | kVA | no | - | The apparent power |
```

Comment:
I am not sure about the multiplier : I think MVA is better.

Reply:
And can you add as a note that for the leg of a three widnings transformer, the normal apparent power shall be identical at both sides (network side and star bus side).

Reply:
I prefer normal apparent power than apparent power only.

Reply:
> I am not sure about the multiplier : I think MVA is better.

The javadoc should probably get changed too.